plugins/scaffolder-backend/src/lib/templating/SecureTemplater.ts

```diff
@@ -87,12 +99,17 @@ const { render, renderCompat } = (() => {
 })();
 `;
 
+export type NunjucksFilter = (...args: JsonValue[]) => JsonValue | undefined;
```

Comment:
I'm quite unsure where to put this. I don't know if this will end up being exported outside `@backstage/plugin-scaffolder-backend` for end users or not. Some advice here would be great.

Reply:
And the failing build on the CI server says no, it is not being exported :)
I'll get that fixed tomorrow.

Reply:
Great, thank you. I'll go ahead and resolve this one.